driver/tests.py

```diff
@@ -72,16 +83,19 @@ def _get_portfolio_configs(portfolio: Path):
         traceback.print_exc()
         raise SyntaxError(
             f"The portfolio {portfolio} could not be loaded.")
-    if "CONFIGS" not in attributes:
-        raise ValueError("portfolios must define CONFIGS")
-    return [config for _, config in attributes["CONFIGS"]]
+    for key, value in attributes.items():
+        # The optimal FDSS 2023 portfolio defines different configs for different PDDL subsets.
+        if key.startswith("CONFIGS"):
+            for _, config in value:
+                yield config
 
 
 def _convert_to_standalone_config(config):
     replacements = [
         ("H_COST_TRANSFORM", "no_transform()"),
         ("S_COST_TYPE", "normal"),
         ("BOUND", "infinity"),
+        ("lpsolver=cplex", "lpsolver=soplex"),  # CPLEX is not available in all CI runners.
```

Comment:
Isn't this unexpected for the user? If we say "we test all parts of all portfolio configurations" and the portfolio uses CPLEX but we test SoPlex instead, the config could still fail with CPLEX.

Reply:
True, I removed this again. Now LP configs are simply skipped on macOS.